### PR TITLE
HPCC-16763 Invalid JSON in language files

### PR DESCRIPTION
**Pull request by miguelvazq**
Whenever there was a extra single quote it caused the widget containing that translation to not load. Replacing single quotes with HTML entities.

Signed-off-by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

File: esp/src/eclwatch/nls/bs/hpcc.js [hunk 1]
```diff
@@ -105,7 +105,7 @@
     DenyRead: "<center>Zabranite<br>Čitanje</center>",
     DenyWrite: "<center>Zabranite<br>Pisanje</center>",
     Depth: "Dubina",
-    DepthTooltip: "'Najveća Dubina Podgrafa",
+    DepthTooltip: "&lsquo;Najveća Dubina Podgrafa",
```
Comment (GordonSmith) [inline esp/src/eclwatch/nls/bs/hpcc.js]: Is this 1 quote deliberate.

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 2]
```diff
@@ -398,7 +398,7 @@ define(
     Query: "Lekérdezés",
     QueryDetailsfor: "A lekérdezés paraméterei",
     QueryID: "Lekérdezés azonosító",
-    QueryIDPlaceholder: "som?q*ry.1'",
+    QueryIDPlaceholder: "som?q*ry.1&rsquo;",
```
Comment (GordonSmith) [inline esp/src/eclwatch/nls/hu/hpcc.js]: I suspect this trailing quote is invalid

**Review by GordonSmith — APPROVED** [review]
Looks ok - question about the single quotes.

File: esp/src/eclwatch/nls/hu/hpcc.js (hunk 2 above)
Comment (richardkchapman) [inline esp/src/eclwatch/nls/hu/hpcc.js]: @AttilaVamos ?

File: esp/src/eclwatch/nls/bs/hpcc.js (hunk 1 above)
Comment (richardkchapman) [inline esp/src/eclwatch/nls/bs/hpcc.js]: @emuharemagic ?

Comment (emuharemagic) [inline esp/src/eclwatch/nls/bs/hpcc.js]: I do not think it is. How do I fix it?

Comment (richardkchapman) [inline esp/src/eclwatch/nls/bs/hpcc.js]: I created a PR to fix it - see https://github.com/hpcc-systems/HPCC-Platform/pull/9426

**Pull request merged**